docs/user/ppl/functions/collection.rst

```diff
@@ -0,0 +1,201 @@
+===========================
+PPL Collection Functions
+===========================
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+   :depth: 1
+
+ARRAY
+-----
+
+Description
+>>>>>>>>>>>
+
+Version: 3.1.0
+
+Usage: ``array(value1, value2, value3...)`` create an array with input values. Currently we don't allow mixture types. We will infer a least restricted type, for example ``array(1, "demo")`` -> ["1", "demo"]
```

Comment:
Question: what is the reason to support `infer a least restricted type` instead of throwing exception?

Reply:
please add a `Limitation: ` after each `Usage: ` to explain these functions only work with `plugins.calcite.enabled=true`

Reply:
> Question: what is the reason to support `infer a least restricted type` instead of throwing exception?

This is aligned with SPARK.